mdk/src/main/resources/pack.mcmeta

```diff
@@ -0,0 +1,11 @@
+# http://minecraft.gamepedia.com/Resource_pack#Contents
+# This will not show up in GuiResourcePackList
+{
+    "pack": {
+        # Unused, but required.
+        "description": "examplemod resources",
+        # A pack_format of "3" should be used starting with Minecraft 1.11. All resources, including language files, should be lowercase (eg: en_us.lang).
+        # A pack_format of "2" will load your mod resources with LegacyV2Adapter, which requires language files to have uppercase letters (eg: en_US.lang).
```

Comment:
JSON doesn't actually support comments, so you can't do them the way you're doing them now. You need to actually have a string field in the JSON file that stores the comment.

e.g.

```json
"_coment": "Whatever"
```